packages/ember-htmlbars/tests/helpers/each_test.js

```diff
@@ -1,617 +0,0 @@
-
-  equal(ulView.$('li').length, 2, 'renders two <li> elements');
-
-  run(function() {
-    people.pushObject({ name: 'Black Francis' });
-  });
-
-  equal(ulView.$('li').length, 3, 'renders an additional <li> element when an object is added');
-
-  runDestroy(ulView);
-});
-
-QUnit.test('it works inside a table element', function() {
-  var tableView = EmberView.create({
-    template: compile('<table><tbody>{{#each view.people}}<tr><td>{{name}}</td></tr>{{/each}}</tbody></table>'),
-    people: people
-  });
-
-  runAppend(tableView);
-
-  equal(tableView.$('td').length, 2, 'renders two <td> elements');
-
-  run(function() {
-    people.pushObject({ name: 'Black Francis' });
-  });
-
-  equal(tableView.$('td').length, 3, 'renders an additional <td> element when an object is added');
-
-  run(function() {
-    people.insertAt(0, { name: 'Kim Deal' });
-  });
-
-  equal(tableView.$('td').length, 4, 'renders an additional <td> when an object is inserted at the beginning of the array');
-
-  runDestroy(tableView);
-});
-
-QUnit.test('it uses {{else}} when replacing model with an empty array', function() {
-  runDestroy(view);
-  view = EmberView.create({
-    [OWNER]: owner,
-    template: compile('{{#each view.items as |item|}}{{item}}{{else}}Nothing{{/each}}'),
-    items: emberA(['one', 'two'])
-  });
-
-  runAppend(view);
-
-  assertHTML(view, 'onetwo');
-
-  run(function() {
-    view.set('items', emberA());
-  });
-
-  assertHTML(view, 'Nothing');
-});
-
-QUnit.test('it uses {{else}} when removing all items in an array', function() {
-  runDestroy(view);
-  var items = emberA(['one', 'two']);
-  view = EmberView.create({
-    template: compile('{{#each view.items as |item|}}{{item}}{{else}}Nothing{{/each}}'),
-    items
-  });
-
-  runAppend(view);
-
-  assertHTML(view, 'onetwo');
-
-  run(function() {
-    items.shiftObject();
-    items.shiftObject();
-  });
-
-  assertHTML(view, 'Nothing');
-});
-
-QUnit.test('it can move to and from {{else}} properly when the backing array gains and looses items (#11140)', function() {
-  runDestroy(view);
-  var items = emberA(['one', 'two']);
-  view = EmberView.create({
-    template: compile('{{#each view.items as |item|}}{{item}}{{else}}Nothing{{/each}}'),
-    items
-  });
-
-  runAppend(view);
-
-  assertHTML(view, 'onetwo');
-
-  run(function() {
-    items.shiftObject();
-    items.shiftObject();
-  });
-
-  assertHTML(view, 'Nothing');
-
-  run(function() {
-    items.pushObject('three');
-    items.pushObject('four');
-  });
-
-  assertHTML(view, 'threefour');
-
-  run(function() {
-    items.shiftObject();
-    items.shiftObject();
-  });
-
-  assertHTML(view, 'Nothing');
-});
-
-QUnit.module('{{each bar as |foo|}}', {
-  setup() {
-    owner = buildOwner();
-    owner.register('view:toplevel', EmberView.extend());
-  },
-  teardown() {
-    runDestroy(owner);
-    runDestroy(view);
-    owner = view = null;
-  }
-});
-
-QUnit.test('#each accepts a name binding', function() {
```

Comment:
Covered by [existing self and parent scope test](https://github.com/emberjs/ember.js/pull/13349/commits/c08e75f3474e8bb995f5f24b837479bf3fca64f8#diff-d3c97c178f7e01e59143b5bf2f84062aR338)